docs/topics/hacking/debugging.rst

```diff
@@ -15,8 +15,19 @@ As with ipdb normally just add a line in your code at the relevant point:
 
     import ipdb; ipdb.set_trace()
 
-Next open a shell on the running web container (requires you to have
-installed docker-utils_ first)::
+Next open a shell on the running web container::
+
+    docker exec -t -i olympia_web_1 supervisorctl fg olympia
```

Comment:
Gah that's is definitely not memorable, but at least it takes you straight to the foregrounded runserver.